c/shared/source/uforead/uforead.c

```diff
@@ -1682,32 +1681,39 @@ static char* getXmlAttrValue(xmlAttr *attr){
     return (char*) attr->children->content;
 }
 
-static void parseXMLLib(ufoCtx h, xmlNodePtr cur) {
+static bool setXMLLib(ufoCtx h, xmlNodePtr cur, char* keyName) {
```

Comment:
You've declared this `bool`, presumably to track whether you ran into some problem parsing (given the check on the call), but you're not returning anything.